cms/djangoapps/contentstore/views/course.py

```diff
@@ -1090,8 +1094,14 @@ def advanced_settings_handler(request, course_key_string):
                     )
 
                     if is_valid:
-                        # update the course tabs if required by any setting changes
-                        _refresh_course_tabs(request, course_module)
+                        try:
+                            # update the course tabs if required by any setting changes
+                            _refresh_course_tabs(request, course_module)
+                        except InvalidTabsException as err:
+                            response_message = [
```

Comment:
Does this get displayed to the user? We might want to have both a developer message and a user message, like we did with the APIs. For example, here's the Team's API:

https://github.com/edx/edx-platform/blob/master/lms/djangoapps/teams/views.py#L519